blockstack/lib/nameset/namedb.py

```diff
@@ -955,12 +955,12 @@ def get_account_history(self, address, offset=None, count=None):
         return namedb_get_account_history(cur, address, offset=offset, count=count)
 
 
-    def get_all_account_addresses(self):
+    def get_all_account_addresses(self, force_allow=False):
         """
         TESTING ONLY
         Get all account addresses
         """
-        assert BLOCKSTACK_TEST, 'BUG: this method can only be accessed in test mode'
+        assert BLOCKSTACK_TEST or force_allow, 'BUG: this method can only be accessed in test mode'
```

Comment:
The reason this method won't run in production is because it can trigger OOMs and stall the node.  But, I see that you need it for the migration CLI option.  Instead of adding `force_allow`, can you add something more descriptive here, like `from_cli` or something, so the reader knows that this is permitted from the CLI but not from, say, the RPC codepath?

Reply:
Fixed 👍